Sources/Afterpay/Wrappers/ObjcWrapper.swift

```diff
@@ -168,6 +170,370 @@ public final class ObjcWrapper: NSObject {
     }
   }
 
-  // swiftlint:enable function_parameter_count
+  // Cash App
+  // swiftlint:disable type_name
+
+  @objc static var cashAppClientId: String {
+    Afterpay.cashAppClientId ?? ""
```

Comment:
Any reason not to make this optional?

Reply:
no reason - great pickup. addressed in [71f61b6](https://github.com/afterpay/sdk-ios/pull/262/commits/71f61b64c75668143e62c26294ed233250623d8a)